confirmation/models.py

```diff
@@ -109,6 +113,9 @@ def get_object_from_key(
     assert obj is not None
 
     used_value = confirmation_settings.STATUS_USED
+    if allow_used and obj.status == used_value:
+        return obj
```

Comment:
This seemed like a nicer way to write this:
``` diff
diff --git a/confirmation/models.py b/confirmation/models.py
index 4e83a829ee..8906aed9b9 100644
--- a/confirmation/models.py
+++ b/confirmation/models.py
@@ -94,6 +94,9 @@ def get_object_from_key(
     confirmation object as used (which generally prevents it from
     being used again). It should always be False for MultiuseInvite
     objects, since they are intended to be used multiple times.
+
+    By default, used confirmation objects cannot be used again as part
+    of their security model.
     """
 
     # Confirmation keys used to be 40 characters
@@ -112,14 +115,13 @@ def get_object_from_key(
     obj = confirmation.content_object
     assert obj is not None
 
-    used_value = confirmation_settings.STATUS_USED
-    if allow_used and obj.status == used_value:
-        return obj
+    forbidden_statuses = {confirmation_settings.STATUS_REVOKED}
+    if not allow_used:
+        forbidden_statuses.add(confirmation_settings.STATUS_USED)
 
-    revoked_value = confirmation_settings.STATUS_REVOKED
-    if hasattr(obj, "status") and obj.status in [used_value, revoked_value]:
+    if hasattr(obj, "status") and obj.status in forbidden_statuses:
         # Confirmations where the object has the status attribute are one-time use
-        # and are marked after being used (or revoked).
+        # and are marked after being revoked (or used).
         raise ConfirmationKeyError(ConfirmationKeyError.EXPIRED)
 
     if mark_object_used:
```